assets/ts/ui/autocomplete/index.ts

```diff
@@ -29,7 +29,10 @@ function getLikelyQueryParams(): string | undefined {
     window.location.search,
     window.decodeURIComponent
   );
-  return searchParams.query || searchParams.name || searchParams.address;
+  const { query, name, address, latitude, longitude } = searchParams;
+  const latlon =
+    latitude && longitude ? `${latitude}, ${longitude}` : undefined;
+  return query || name || address || latlon;
```

Comment:
Adding the latitude/longitude here gives us something to display in the input field when using your geolocated location.